lib/http.js

```diff
@@ -255,12 +262,6 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
           'DataError',
           {account, httpStatusCode: 400, public: true});
       }
-      if(!Array.isArray(zcaps)) {
-        throw new BedrockError(
-          'The "zcaps" property must be an array in the body.',
-          'DataError',
-          {account, httpStatusCode: 400, public: true});
-      }
       // TODO: Add permissions to allow access for admin
       if(account !== accountId) {
```

Comment:
And this will end up changing to use zcaps, right?

Reply:
Not sure what you're referring to?  Is it another new feature.  Does it need to hold up this PR?

Reply:
We would be using http-sigs and zcaps for authorization instead of this mock check. This should not hold up the PR and should be apart of a future iteration of this lib.

Reply:
Agree with @gannan08 but it shouldn't hold up the PR.